Public/Src/Engine/Processes/SandboxedProcess.cs

```diff
@@ -197,10 +196,59 @@ public async Task<SandboxedProcessResult> GetResultAsync()
                     return null;
                 }
 
-                return detouredProcess.GetJobObject()?.GetPeakMemoryUsage();
+                ulong? currentPeakWorkingSet = null;

```

Comment:
Why `ulong?` with `null` as default instead of `ulong` with `0` as default?  It seems the same to me and it would make some of the code below syntactically simpler

Reply:
It is more related to the callers of that method. In potential future scenarios, we would want to understand whether we were able to get the memory usage at all. However, 0 can mean that as well. I will think more

---
In reply to: [337179878](https://github.com/microsoft/BuildXL/pull/1076#discussion_r337179878) [](ancestors = 337179878)